x-pack/plugin/eql/src/main/resources/org/elasticsearch/xpack/eql/plugin/eql_whitelist.txt

```diff
@@ -18,6 +18,11 @@ class org.elasticsearch.xpack.ql.expression.function.scalar.whitelist.InternalQl
   double nullSafeSortNumeric(Number)
   String nullSafeSortString(Object)
 
+#
```

Comment:
Why did you add an extra ASCII Functions block?
https://github.com/elastic/elasticsearch/pull/56404/files#diff-610841f585d6bbff1fd42fb91c3c424aR65

Reply:
This ASCII block is under a different ScriptUtils class. The method comes via QL, whereas the other ASCII block handles EQL specific functions.

Reply:
Sorry, missed that.